src/components/crossDevice/ClientSuccess/index.js

```diff
@@ -15,14 +15,12 @@ class ClientSuccess extends Component {
     return (
       <div>
         <PageTitle
-          title={translate('cross_device.client_success.title')}
-          subTitle={translate('cross_device.client_success.sub_title')}
+          title={translate('xdevice_return.title')}
+          subTitle={translate('xdevice_return.subtitle')}
```

Comment:
As long as we're consistent with all cross-device screens, it should be fine, no? Could we add a note in the documentation to explain this?

Reply:
Yes, I agree with Kimmi on the fact that integrators might be more familiar with "cross_device". I also prefer more explicit naming in general.

Reply:
@raemarielee so far only Web SDK uses `xdevice_*` keys right? I think it makes sense to use `cross_device` instead of `xdevice` because it could be ambiguous to refer to a flow.

Reply:
Yeap, it's a Web SDK specific flow